VideoLocker/res/menu/edit_profile.xml

```diff
@@ -2,6 +2,6 @@
 <menu xmlns:android="http://schemas.android.com/apk/res/android">
     <item
         android:id="@+id/edit_profile"
-        android:showAsAction="always"
+        android:showAsAction="always|withText"
```

Comment:
Not sure where that default is coming from, but I set the size manually to edx_base (16sp) which is a bit bigger.